### ISSUE TITLE
setting maxconn is not possible in defaults section via default-server

### ISSUE THREAD

**Issue by sigmonsays** [haproxytech/dataplaneapi#139]
Using latest build from github of dataplaneapi

The following curl command (1) results in the following haproxy snippet below (2)
```
curl -X PUT --user $user \
-H "Content-Type: application/json" \
-d '{
 "default_server": {    
   "maxconn": 500
 }
}' "$service/v2/services/haproxy/configuration/defaults?version=$_version"

```

haproxy config snippet after curl command
```
defaults 
  log global
  default-server 
```

**Comment by prometherion** [comment]
Hi @sigmonsays, the last commit provided a `client-native` bump-up and the issue was fixed, implemented in [`ceedbe0`](https://github.com/haproxytech/client-native/commit/ceedbe03c9605dff772a7660a8fb3a80a962ea21).

Please, test again building from the current [`master`](https://github.com/haproxytech/dataplaneapi/commit/1c1b3cec46934d3f3c795639bad19194406ee1fc) and feel free to reach us for further issues!

**Comment by sigmonsays** [comment]
@prometherion thanks!

I just tested on latest master and can now set maxconn properly. Thanks for the quick fix!
